core/src/main/java/org/mskcc/cbio/oncokb/model/Alteration.java

```diff
@@ -43,6 +47,32 @@ public void setPortalAlterations(Set<PortalAlteration> portalAlterations) {
     public Alteration() {
     }
 
+    public Alteration(String hgvs) {
+        if (hgvs != null && !hgvs.trim().isEmpty()) {
+            TranscriptConsequence transcriptConsequence = GenomeNexusUtils.getTranscriptConsequence(hgvs);
+            if (transcriptConsequence != null) {
+                String hugoSymbol = transcriptConsequence.getGeneSymbol();
```

Comment:
`GenomeNexusUtils.getTranscriptConsequence` will pick the one match with our transcript